bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/TreeItem.java

```diff
@@ -375,7 +375,7 @@ public Color getBackground (int index) {
  */
 public Rectangle getBounds () {
 	checkWidget ();
-	return DPIUtil.autoScaleDown(getBoundsInPixels());
+	return DPIUtil.scaleDown(getBoundsInPixels(), parent.getZoom());
```

Comment:
Why is `getZoom()` called on the parent at some places (e.g., `TreeItem`, `TreeColum`, `Tracker`, ...) instead of just calling `getZoom()` on the widget iself?

Reply:
Changed back to getZoom()